client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -104,9 +104,9 @@ class LoggedInForm extends Component {
 			authorizeError
 		} = props.jetpackConnectAuthorize;
 
-		// For SSO or WooCommerce Services users, do not display plans page
+		// For SSO, WooCommerce Services, and JPO users, do not display plans page
 		// Instead, redirect back to admin as soon as we're connected
-		if ( props.isSSO || props.isWCS ) {
+		if ( props.isSSO || props.isWCS || ( 'undefined' !== typeof queryObject && 'jpo' === queryObject.from ) ) {
```

Comment:
Can it be 
`props.isSSO || props.isWCS || ( queryObject && 'jpo' === queryObject.from )`